spec/factories/cuisine.rb

```diff
@@ -0,0 +1,16 @@
+FactoryBot.define do
+
+  factory :cuisine do
+    name { Faker::Address.state + "#{rand(10000)}" }
```

Comment:
If the intention is to have a unique name, then FactoryBot's `sequence` feature might be a better choice?